changelog/1727.trivial.rst

```diff
@@ -0,0 +1 @@
+Bump minimum astropy version to 5.0.1
```

Comment:
```suggestion
Increased minimum version of Astropy_ to 5.0.
```

Just to remain consistent with similar past entries and our [changelog guide](https://docs.plasmapy.org/en/latest/contributing/changelog_guide.html#changelog-guidelines) (i.e. use of past tense).  The ``Astropy_`` is an explicit link that will generate a link to https://www.astropy.org/.